doc/WebSite/Multi-Lingual-Entities.md

```diff
@@ -0,0 +1,89 @@
+### Introduction
+
+ASP.NET Boilerplate provides two basic interfaces for Multi-Lingual entity definitions. 
+
+#### IMultiLingualEntity
+
+IMultiLingualEntity interface represents is used to mark multi lingual entities. The entities marked with IMultiLingualEntity interface must define language-neutral information. The entities marked with IMultiLingualEntity contains a collection of Translations which contains language-dependent information.
+
+A sample multi lingual entity would be;
+
+```c#
+public class Product : IMultiLingualEntity<ProductTranslation>
+{
+    public decimal Price {get; set;} 
+    
+    public virtual ICollection<ProductTranslation> Translations { get; set; }
+}
+```
+
+#### IEntityTranslation
+
+IEntityTranslation interface is used to mark translation of a Multi-Lingual entity. The entities marked with IEntityTranslation interface must define language dependent information. The entities marked with IEntityTranslation contains Language field which contains a language code for the translation and a reference to Multi-Lingual entity.
+
+A sample multi lingual entity would be;
+
+```c#
+public class ProductTranslation : Entity, IEntityTranslation<Product>
+{
+	public string Name { get; set; }
+
+	public Product Core { get; set; }
+
+	public int CoreId { get; set; }
+
+	public string Language { get; set; }
+}
+```
+
+ #### CreateMultiLingualMap 
+
+CreateMultiLingualMap is an extension method to map a Multi-Lingual entity and one of it's Translations to a appropriate Dto class. 
```

Comment:
This section should be clear about that it's using AutoMapper. Also, it would be better to define the purpose as the first sentence.